test/services/carrier_account.test.js

```diff
@@ -76,22 +101,50 @@ describe('CarrierAccount Service', function () {
   });
 
   it('updates a carrier account', async function () {
+    // Create a temporary carrier account
+    const carrierAccount = await this.client.CarrierAccount.create(Fixture.basicCarrierAccount());
+
     const testDescription = 'My custom description';
+    const updateParams = {
+      description: testDescription,
+    };
 
-    const carrierAccount = await this.client.CarrierAccount.create(Fixture.basicCarrierAccount());
+    // Update the carrier account
```

Comment:
style: let's not include comments that say what the next line does when it's easy to see that `create()` creates a record and so on. The only real comment that makes sense to keep is the one detailing why we need to delete a record in a test context when it's not the thing being tested. The other comments should be avoided.